benchmark/napi/function_args/index.js

```diff
@@ -12,15 +12,29 @@ let napi;
 try {
   v8 = require(`./build/${common.buildType}/binding`);
 } catch {
-  console.error(`${__filename}: V8 Binding failed to load`);
-  process.exit(0);
+  try {
+    common.buildAddon('benchmark/napi/function_args');
+    // eslint-disable-next-line node-core/no-duplicate-requires
+    v8 = require(`./build/${common.buildType}/binding`);
+  } catch (e) {
+    console.error(`${__filename}: V8 Binding failed to load`);
+    console.error(e);
+    process.exit(0);
+  }
```

Comment:
As a suggestion: I would rewrite this whole part to something like:

```js
const v8 = common.buildAddon(
  `./build/${common.buildType}/binding`,
  'benchmark/napi/function_args'
);
```

In that case the common call just tries to require it first and returns it if possible and tries to build the addon otherwise.
If it fails, we also already have all the necessary information printed due to the `console.error` part in the common part and in the actual error that is thrown if it fails.

This would reduce the number of code lines significantly.